### PR TITLE
feat: display event form popup in month

### PR DESCRIPTION
**Pull request by lja1018**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- display event form popup when grid selection is created in month view

https://user-images.githubusercontent.com/28647773/147800997-2bcc2dea-6f1b-45dc-a587-a8367223ea78.mov

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: apps/calendar/src/css/popup/common.css [hunk 1]
```diff
@@ -33,7 +33,7 @@
 
 .dropdown-menu {
   position: absolute;
-  width: 100%;
+  width: calc(100% - 2px);
```
Comment (jwlee1108) [inline apps/calendar/src/css/popup/common.css:36]: -2px은 어떤건가요?

**Review by jwlee1108 — COMMENTED** [review]
리뷰 완료합니다.

File: apps/calendar/src/css/popup/common.css (hunk 1 above)
Comment (lja1018) [reply apps/calendar/src/css/popup/common.css:36]: 원래는 border의 양쪽 너비를 빼준건데 생각해보니 `box-sizing: border-box;`가 floating layer에는 적용되지 않고 있었네요.
다시 롤백하고 floating layer에 `box-sizing: border-box;` 적용했습니다.

File: apps/calendar/src/hooks/dayGridCommon/popupWithDayGridSelection.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import { compare } from '@src/time/datetime';
+
+interface Params {
+  gridSelection: GridSelectionData | null;
+  useCreationPopup: boolean;
+  dateMatrix: TZDate[][];
+}
+
+function sortDate(start: TZDate, end: TZDate) {
+  const isIncreased = compare(start, end) === -1;
```
Comment (adhrinae) [inline apps/calendar/src/hooks/dayGridCommon/popupWithDayGridSelection.ts:18]: `compare` 함수의 내부 동작에 의존하고 있는데, 결국은 나중에 이 함수에 익숙지 않은 사람은 `compare` 함수 내부를 들여다봐야 하네요.
단순히 `start < end` 로 비교하는건 안되나요?

**Review by adhrinae — APPROVED** [review]
리뷰 완료합니다.

File: apps/calendar/src/hooks/dayGridCommon/popupWithDayGridSelection.ts (hunk 2 above)
Comment (lja1018) [reply apps/calendar/src/hooks/dayGridCommon/popupWithDayGridSelection.ts]: `start < end`로 단순 비교하도록 수정했습니다.

**Pull request merged**